readthedocs/doc_builder/python_environments.py

```diff
@@ -215,8 +215,10 @@ def setup_base(self):
             site_packages,
             '--no-download',
             env_path,
-            bin_path=None,  # Don't use virtualenv bin that doesn't exist yet
-            cwd=self.checkout_path,
+            # Don't use virtualenv bin that doesn't exist yet
+            bin_path=None,
+            # Don't use the project's root, some config files can interfere
+            cwd='$HOME',
```

Comment:
I wonder if we should be running this from `/tmp` or something else that isn't in the checkout. I guess we aren't always sure any other path will exist (unless we make one w/ the tempfile module)`, so this seems fine. But I do wonder if we'll hit other issues at some point (eg. if we add our own `pyproject.toml` file for RTD?

Reply:
Do the docker containers have access to the rtd code? I don't think so. But, yeah, using `/tmp` sounds better. People running this outside docker could have some problems I think, but I'm less worried about that. Should I go for plain `/tmp` or create one with the tempfile module?

Reply:
Ah good point, I forgot about Docker not having access. I think we can just go with `$HOME` for now, because it feels overkill (and could lead to errors) to create a temp directory.